code/src/components/StartScreen.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { Player } from '@lottiefiles/react-lottie-player';
+import UserNameInput from './UserNameInput';
+import GameScreen from './GameScreen';
+import Loading from './Loading';
+import GameScreenLotties from './GameScreenLotties';
+
```

Comment:
Engaging use of the lotties, fitting of the theme and leading the user what to expect. We think using the lotties in more than one place made it super pleasing to the eye.